### PR TITLE
Run builds with Gradle 7

### PR DESCRIPTION
**Pull request by ferozco**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -24,8 +24,9 @@ buildscript {
         classpath 'com.gradle.publish:plugin-publish-plugin:0.14.0'
```
Comment (robert3005) [inline build.gradle]: bump this to 0.15.0

File: build.gradle [hunk 2]
```diff
@@ -24,8 +24,9 @@ buildscript {
...
         classpath 'com.netflix.nebula:gradle-info-plugin:9.3.0'
```
Comment (robert3005) [inline build.gradle]: bump this to 10.0.0

File: gradle-baseline-typescript/build.gradle [hunk 3]
```diff
@@ -57,6 +57,10 @@ gradlePlugin {
     }
 }
 
+tasks.named('processResources').configure {
+    duplicatesStrategy 'include'
```
Comment (robert3005) [inline gradle-baseline-typescript/build.gradle:61]: do you know where the conflicts are coming from?

Comment (ferozco) [reply gradle-baseline-typescript/build.gradle:61]: this is coming from the resource we explicitly add (in the resource directory) and one generated by the gradle-plugin plugin. We need the explicit resource for testing to work and the plugin also manages publishing

**Review by robert3005 — APPROVED** [review]

**Comment by fawind** [comment]
Nice, was just looking for this! Can we get a new release that includes this PR?

**Comment by robert3005** [comment]
there's 1.3.0 released

**Pull request merged**